modules/virt-attaching-vm-to-sriov-network.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.adoc
+
+[id="virt-attaching-vm-to-sriov-network_{context}"]
+= Attaching a virtual machine to an SR-IOV network
+
+You can attach the virtual machine to the SR-IOV network by including the network details in the virtual machine configuration. 
+
+.Procedure
+
+. Include the SR-IOV network details in the `spec.domain.devices.interfaces` and `spec.networks` of the virtual machine configuration:
++
+[source,yaml]
+----
+kind: VirtualMachine
+...
+spec:
+  domain:
+    devices:
+      interfaces:
+      - masquerade: {} <1>
+        name: default
+      - name: <sriov-network> <2>
+        sriov: {}
+  networks:
+  - name: default <3>
+    pod: {}
+  - multus:
+    name: <sriov-network> <4>
+...
+----
+<1> The `masquerade` binding to the default Pod network.
+<2> The name of the SR-IOV network.
+<3> The default Pod network.
+<4> The name of the SR-IOV network.
```

Comment:
This needs to be the same in both 2 and 4, right? If it's actually a requirement, I might state that.